src/components/UI/Text/InputNumber.vue

```diff
@@ -32,9 +32,16 @@
         <inline-svg :src="startIconObj"></inline-svg>
       </div>
 
+      <!-- static text -->
+      <div v-if="isStaticTextEnabled" :class="[staticText.class, staticTextDefaultClass]">
+        <p class="self-center">
+          {{ staticText.text }}
+        </p>
+      </div>
```

Comment:
not sure if this should be added here? Feels too specific? might be better to be added separately wherever it is used?